compose/docker-compose.yml

```diff
@@ -47,7 +47,7 @@ services:
       start_period: 30s
     restart: on-failure
   keystone-init:
-    image: registry.cn-beijing.aliyuncs.com/yunionio/compose-service-init:v3.11.9
+    image: ${CLOUDPODS_REPO:-registry.cn-beijing.aliyuncs.com/yunionio}/compose-service-init:v4-k3s.4
```

Comment:
The version of v4-k3s.4 is wrong.

Reply:
Yes, I noticed but was not sure if that is by design. PR updated.